crates/rspack/tests/tree-shaking/pure_comments_magic_comments/snapshot/snap.diff

```diff
@@ -0,0 +1,32 @@
+--- expected
++++ actual
+@@ -3,27 +3,18 @@
+ ---
+ ```js title=main.js
+ (self['webpackChunkwebpack'] = self['webpackChunkwebpack'] || []).push([["main"], {
+-"./answer.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
+-"use strict";
+-__webpack_require__.r(__webpack_exports__);
+-__webpack_require__.d(__webpack_exports__, {
+-  answer: function() { return answer; }
+-});
+- const answer = 103330;
+-}),
+ "./app.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
+ "use strict";
+ __webpack_require__.r(__webpack_exports__);
+ __webpack_require__.d(__webpack_exports__, {
+   a: function() { return a; }
+ });
+-/* harmony import */var _answer__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/* ./answer */"./answer.js");
+ 
+ function res() {
+-    _answer__WEBPACK_IMPORTED_MODULE_0__.answer;
++    answer;
+ }
+  const a = 3;
+-var __WEBPACK_DEFAULT_EXPORT__ = res();
```

Comment:
The difference is due to the previous tree shaking doesn't implement pure expression.